src/ontology/mondo-edit.obo

```diff
@@ -314658,10 +314657,13 @@ property_value: confidence "0.23456790123456783" xsd:double
 
 [Term]
 id: MONDO:0016343
-name: unclassified cardiomyopathy
+name: obsolete unclassified cardiomyopathy
 subset: ordo_group_of_disorders {source="Orphanet:217678"}
 xref: Orphanet:217678 {source="MONDO:equivalentTo"}
-is_a: MONDO:0004994 {source="Orphanet:217678"} ! cardiomyopathy
+property_value: IAO:0000231 "out of scope" xsd:string
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2824 xsd:string
+is_obsolete: true
+consider: cardiomyopathy
```

Comment:
this should be the ID: MONDO:0004994

You can either add the ID directly, or add the IRI, and then it will display the label with a hyperlink, but it will still look like consider: MONDO:0004994 here.

Instructions:
1. Search for the 'cardiomyopathy' class
2. Click on it in the hierarchy and type command + U
3. Check the box, 'show full IRI'
4. Copy the IRI
5. Go to your obsolete class
6. Add annotation: consider, and select the IRI editor tab
7. paste in the IRI
8. Click OK

Reply:
Consider annotation has now got the full IRI